extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -40,3 +40,13 @@ amp-sticky-ad {
 .-amp-sticky-ad-layout > amp-ad {
   display: block;
 }
+
+.-amp-sticky-ad-close-button {
+  position: absolute;
```

Comment:
Yes it would. I set `amp-sticky-ad` to `overflow: visible !important` to avoid this. I tried different `amp-ad` width and height here. Close button is always visible.